spec/ParseQuery.spec.js

```diff
@@ -4318,6 +4318,40 @@ describe('Parse.Query testing', () => {
     }
   });
 
+  it_only_db('mongo')('deeply nested Pointers (issue #7413)', async function (done) {
+    const parent = new Parse.Object('Parent');
+    const child1 = new Parse.Object('Child');
+    const child2 = new Parse.Object('Child');
+
+    parent.set('subDocument', { child: child1 });
+    parent.set('children', [
+      { child: child1, count: 2 },
+      { child: child2, count: 3 },
+    ]);
+    await parent.save();
+
+    // equalTo
```

Comment:
Can you please remove these comments, because the tests seem pretty simple and self-explanatory.